ProjFS.Mac/PrjFSKextTests/MockVnodeAndMount.hpp

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include "../PrjFSKext/kernel-header-wrappers/vnode.h"
+#include "../PrjFSKext/kernel-header-wrappers/mount.h"
+#include "../PrjFSKext/public/FsidInode.h"
+#include <memory>
+#include <string>
+
+typedef std::shared_ptr<mount> MountPointer;
+typedef std::shared_ptr<vnode> VnodePointer;
+typedef std::weak_ptr<vnode> VnodeWeakPointer;
+
+struct mount
+{
+private:
+    vfsstatfs statfs;
+    uint64_t nextInode;
+    
+public:
+    static MountPointer Create(const char* fileSystemTypeName, fsid_t fsid, uint64_t initialInode);
+    
+    inline fsid_t GetFsid() const { return this->statfs.f_fsid; }
```

Comment:
Nit: There is no need for `inline` here and we're not using it for the functions below (in `vnode`)